docs/contributing/doc_guide.rst

```diff
@@ -1112,24 +1129,92 @@ Narrative documentation guidelines
+in a docstring. Sphinx attempts to link words in type specifications to
+code objects. Type lines are intended to provide concise information
+about allowed types, sizes, shapes, physical types, and default values
+of a parameter. To resolve this warning, first move information about
+the *meaning* of a parameter from the type specification into the
+parameter description that begins on the following line. To expand the
+list of allowed words or patterns in type specifications, add a regular
+expression to ``nitpick_ignore_regex`` in |docs/conf.py|_.
 
-  - If the text should refer to a code object, make sure that the full
-    namespace is provided correctly.
+Missing documentation pages for new modules
+-------------------------------------------
 
-  - If the text is in the type line for a parameter in a docstring, then
-    either:
+When a new module or subpackage is created, it is usually necessary to
+:ref:`create a stub file <api-static>` for it in |docs/api_static|_.
 
-    - Change the type line to include only Python types and words or
-      patterns defined in ``nitpick_ignore_regex`` in |docs/conf.py|_
-      (preferred), or
+Missing attribute errors
+------------------------
+
+An `AttributeError` may occur when an ``import`` statement is missing in
+a :file:`__init__.py` file.  For example, the error
+
+```
+AttributeError: module 'plasmapy.subpackage' has no attribute 'module'
+```
+
+will occur when :file:`plasmapy/subpackage/__init__.py` is missing
+:py:`from plasmapy.subpackage import module`. Make sure that ``__all__``
+contains ``"module"`` as well.
+
+Errors that are unrelated to a pull request
+-------------------------------------------
+
+Occasionally, documentation builds will start failing for reasons that
+have nothing to do with the changes made in a pull request. Such errors
+generally result from a new release of a package that is required for
+PlasmaPy's documentation build.
+
+.. tip::
+
+   If you are a new contributor and have encountered a strange
+   documentation build failure, first check recent issues_ to see if one
+   has already been created about it. If an issue has not already been
+   created, please `raise an issue about the documentation build
+   failure`_.
+
+To figure out if a new release caused the error, search PyPI_ for
+recently released packages, including `packages related to Sphinx`_ and
+any that came up in the error message. You can also check if the same
+documentation build failure happened in the `weekly tests`_ on the
+``main`` branch. After identifying the package that caused the error, a
+pull request can be submitted that sets a temporary maximum allowed
+version of the package that can be revisited later.
+
+.. tip::
 
-    - Add the text as a regular expression to ``nitpick_ignore_regex``
-      so that Sphinx knows not to issue a warning for this text.
+   When dealing with this kind of error, procrastination often pays off!
```

Comment:
```suggestion
   Procrastination often pays off!
```

Only kidding, of course. Or am I...

Reply:
This is one of the exceptionally rare comments for which every GitHub emoji response is equally valid.